Resources/Prototypes/Entities/Clothing/Head/hats.yml

```diff
@@ -467,3 +467,14 @@
     sprite: Clothing/Head/Hats/nursehat.rsi
   - type: Clothing
     sprite: Clothing/Head/Hats/nursehat.rsi
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadBeretBlueShield
+  name: beret blue shield
+  description: ...
```

Comment:
1. `blue shield's beret` flows better
2. Is it so hard to write a description?

Reply:
Not all at once, I have to discuss the descriptions with my comrades. We try to do it with humor

Reply:
better remove both name and description tags and add corresponding entries to an .ftl file. Otherwise your comrades will have to read these unlocalized